libraries/AP_HAL_ChibiOS/hwdef/HolybroF4_PMU/hwdef.dat

```diff
@@ -0,0 +1,117 @@
+# hw definition file for f405 Holybro CAN PMU
+
+# MCU class and specific type
+MCU STM32F4xx STM32F405xx
+
+# bootloader starts firmware at 64k
+FLASH_RESERVE_START_KB 64
+FLASH_SIZE_KB 1024
+
+# store parameters in pages 2 and 3
+STORAGE_FLASH_PAGE 2
+define HAL_STORAGE_SIZE 15360
+
+# board ID for firmware load
+APJ_BOARD_ID 5401
+
+env AP_PERIPH 1
+
+define STM32_ST_USE_TIMER 5
+define CH_CFG_ST_RESOLUTION 32
+
+# enable watchdog
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+STDOUT_SERIAL SD3
+STDOUT_BAUDRATE 57600
+
+# blue LED0 marked as ACT
+PB0 LED OUTPUT HIGH
+define HAL_LED_ON 0

```

Comment:
I don't think this LED will blink at the moment.  You need to set things like this now:
define AP_NOTIFY_GPIO_LED_1_ENABLED 1
define AP_NOTIFY_GPIO_LED_1_PIN 90